bc_obps/reporting/tests/service/test_report_product_service.py

```diff
@@ -1,14 +1,13 @@
+from django.test import TestCase
 import pytest
 from registration.models.regulated_product import RegulatedProduct
 from reporting.models.report_product import ReportProduct
 from reporting.service.report_product_service import ReportProductService
 from model_bakery.baker import make_recipe
 
-pytestmark = pytest.mark.django_db
 
-
-class TestReportProductService:
-    def setup_method(self):
+class TestReportProductService(TestCase):
```

Comment:
I had to change the test from pytest to django to be able to use "assertQuerySetEquals"